src/main/java/subway/dto/ShortestPathResponse.java

```diff
@@ -0,0 +1,36 @@
+package subway.dto;
+
+import java.util.List;
+import subway.domain.Path;
+
+public class ShortestPathResponse {
+
+    private List<String> path;
+    private int distance;
+    private int fare;
+
+    private ShortestPathResponse() {
+    }
+
+    public ShortestPathResponse(final List<String> path, final int distance, final int fare) {
+        this.path = path;
+        this.distance = distance;
+        this.fare = fare;
+    }
+
+    public static ShortestPathResponse of(final Path path, final int fare) {
+        return new ShortestPathResponse(path.getAllStationName(), path.getDistance().getValue(), fare);
```

Comment:
```suggestion
        return new ShortestPathResponse(path.getAllStationName(), path.getDistance(), fare);
```
이런식으로 디미터의 법칙을 지켜야 할거 같아요!

Reply:
수정했습니다!